analysis/bulk-deconvolution/exploratory-notebooks/compare-epic-quantiseq.Rmd

```diff
@@ -0,0 +1,441 @@
+* For project `SCPCP000006` (wilms):
+	* CD8 T cells as well as monocytes dominate the `EPIC` results, and while we don't see CD8 as commonly in `quanTIseq`, we do also see monocytes although often in _different_ samples from where this cell type is common according to `EPIC`
+	* `EPIC` results here have fewer cell types in general compared to `quanTIseq`, even considering the difference in references
+* For project `SCPCP000009` (cns tumors):
+	* We see similar patterns to `SCPCP000001` and `SCPCP000002` where NK cells are commonly identified by `quanTIseq` but not by `EPIC`,  `EPIC` estimates substantial portions of endothelial cells, and `quanTIseq` estimates substantial proportions of dendritic cells
+	* We also see T cells, in particular CD4, dominating in `EPIC` but not in `quanTIseq`
+	* In the sample where `EPIC` classifies nearly 100% of the cells, there are similar cell types between methods but vastly different proportions
+* For project `SCPCP000017` (osteo):
+	* Monocytes and CD4 T cells dominate in `EPIC` but these are much less common in `quanTIseq
+	* In the sample where `EPIC` classifies nearly 100% of the cells, the cell types dominating in `EPIC` tend not to be those dominating in `quanTIseq`.
+
+
+
+## Proportions of classified cells
+
+Next, we'll look specifically at the distributions of the total classified proportion (i.e., 1 - "other") across samples.
+For this, we'll only consider samples which are not essentially fully classified.
+There are two samples which `EPIC` fully classified:
+```{r}
+fully_classified_df <- consensus_df |>
+  dplyr::filter(
+    celltype_consensus == "Other", 
+    proportion <= 0.01
+  ) 
+fully_classified_df
+```
+
+
+First, we'll visualize these distributions as sina plots:
+
+```{r, fig.width = 7, fig.height = 5}
+classified_prop_df <- consensus_df |>
+  dplyr::filter(
+    !(sample_id %in% fully_classified_df$sample_id), 
+    celltype_consensus == "Other"
+  ) |>
+  dplyr::mutate(
+    classified_proportion = 1 - proportion
+  )
+
+ggplot(classified_prop_df) + 
+  aes(x = project_name, y = classified_proportion, color = method) + 
+  ggforce::geom_sina(size = 1, position = position_dodge(width = 0.4)) +
+  scale_y_continuous(
+    limits = c(0, 0.6),
+    breaks = seq(0, 0.6, 0.1)
+  ) + 
+  scale_color_brewer(palette = "Dark2") +
+  theme(axis.text.x = element_text(angle = 30, hjust = 1))
+```
+
+Distributions of classified proportions, across all projects, are generally below 0.4 with a handful of outliers. 
+As noted in the barplots, `EPIC` tends to classify lower proportions of each sample compared to `quanTIseq`, but whether this is due to a) the method itself, b) fewer cell types in the `EPIC` signature gene lists, or c) marker gene information in the signature gene matrices themselves would require further investigation.
+
+
+Next, we can ask whether there is any relationship between these quantities:
```

Comment:
Given the differences in references, I'm not sure I would expect any relationship between the other quantities in the two methods. But I might expect to see a relationship between the shared cell types. You don't have to do this, but I do think it would be helpful to have a plot that directly compares the fractions of each of the shared cell types, either using a grouped sina plot or a scatter plot like the one below for each cell type.

Reply:
>  But I might expect to see a relationship between the shared cell types.